ods_ci/tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml

```diff
@@ -34,15 +34,17 @@ items:
       baseDomain: {{ base_domain }}
       compute:
       - name: worker
+        architecture: {{ aws_worker_architecture|default("amd64") }}
         platform:
           aws:
-            type: {{ worker_node_instance_type|default("m5.2xlarge") }}
+            type: {{ worker_node_instance_type }}
         replicas: {{ worker_node_replicas|default("2") }}
       controlPlane:
         name: master
+        architecture: {{ aws_master_architecture|default("amd64") }}
         platform:
           aws:
-            type: {{ master_node_instance_type|default("m5.2xlarge") }}
+            type: {{ master_node_instance_type }}
```

Comment:
[infrastructure_configuration_variables.md](https://github.com/red-hat-data-services/ods-ci/pull/2624/files#diff-4bdf8f96d980a99f14a4d43a44a37387dca385ca2feef04f9048f9adf998b1bc) lists that `master_node_instance_type` and `worker_node_instance_type` have m5.xlarge as defaults. I think we can leave the defaults as they were before, or?

Reply:
Thanks for catching that.